frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.tsx

```diff
@@ -11,14 +12,21 @@ function LevelMissionInfoBanner({
 	openOverlay: () => void;
 }) {
 	return (
-		<button className="level-mission-info-banner" onClick={openOverlay}>
-			<span className="level-title-area">{`Level ${currentLevel + 1}`}</span>
+		<span className="level-mission-info-banner">
+			<h2 className="level-title-area">{`Level ${currentLevel + 1}`}</h2>
 			<p
+				data-testid="banner-info"
 				dangerouslySetInnerHTML={{
 					__html: LEVELS[currentLevel].missionInfoShort ?? '',
 				}}
 			></p>
-		</button>
+			<ThemedButton onClick={openOverlay}>
+				<p className="info-icon" aria-hidden="true">
```

Comment:
a tiny semantic improvement would be to switch this `<p>` tag to a `<span>` given it isn't containing a chunk of text.

Reply:
okay sounds good